Source/DafnyCore/DafnyPrelude.bpl

```diff
@@ -825,283 +825,283 @@ function Math#clip(a: int): int;
 }
 
 // conversion produces a good map.
-axiom (forall<T> s: Seq T :: { MultiSet#FromSeq(s) } $IsGoodMultiSet(MultiSet#FromSeq(s)) );
+axiom (forall s: Seq :: { MultiSet#FromSeq(s) } $IsGoodMultiSet(MultiSet#FromSeq(s)) );
 // cardinality axiom
-axiom (forall<T> s: Seq T ::
+axiom (forall s: Seq ::
   { MultiSet#Card(MultiSet#FromSeq(s)) }
     MultiSet#Card(MultiSet#FromSeq(s)) == Seq#Length(s));
 // building axiom
-axiom (forall<T> s: Seq T, v: T ::
+axiom (forall s: Seq, v: Box ::
   { MultiSet#FromSeq(Seq#Build(s, v)) }
     MultiSet#FromSeq(Seq#Build(s, v)) == MultiSet#UnionOne(MultiSet#FromSeq(s), v)
   );
 
 // concatenation axiom
-axiom (forall<T> a: Seq T, b: Seq T ::
+axiom (forall a: Seq, b: Seq ::
   { MultiSet#FromSeq(Seq#Append(a, b)) }
     MultiSet#FromSeq(Seq#Append(a, b)) == MultiSet#Union(MultiSet#FromSeq(a), MultiSet#FromSeq(b)) );
 
 // update axiom
-axiom (forall<T> s: Seq T, i: int, v: T, x: T ::
+axiom (forall s: Seq, i: int, v: Box, x: Box ::
   { MultiSet#FromSeq(Seq#Update(s, i, v))[x] }
     0 <= i && i < Seq#Length(s) ==>
     MultiSet#FromSeq(Seq#Update(s, i, v))[x] ==
       MultiSet#Union(MultiSet#Difference(MultiSet#FromSeq(s), MultiSet#Singleton(Seq#Index(s,i))), MultiSet#Singleton(v))[x] );
   // i.e. MS(Update(s, i, v)) == MS(s) - {{s[i]}} + {{v}}
-axiom (forall<T> s: Seq T, x: T :: { MultiSet#FromSeq(s)[x] }
+axiom (forall s: Seq, x: Box :: { MultiSet#FromSeq(s)[x] }
   (exists i : int :: { Seq#Index(s,i) } 0 <= i && i < Seq#Length(s) && x == Seq#Index(s,i)) <==> 0 < MultiSet#FromSeq(s)[x] );
 
 // ---------------------------------------------------------------
 // -- Axiomatization of sequences --------------------------------
 // ---------------------------------------------------------------
 
-type Seq T;
+type Seq;
 
-function Seq#Length<T>(Seq T): int;
-axiom (forall<T> s: Seq T :: { Seq#Length(s) } 0 <= Seq#Length(s));
+function Seq#Length(Seq): int;
+axiom (forall s: Seq :: { Seq#Length(s) } 0 <= Seq#Length(s));
 
-function Seq#Empty<T>(): Seq T uses {
-  axiom (forall<T> :: { Seq#Empty(): Seq T } Seq#Length(Seq#Empty(): Seq T) == 0);
+function Seq#Empty(): Seq uses {
+  axiom (Seq#Length(Seq#Empty(): Seq) == 0);
 }
-axiom (forall<T> s: Seq T :: { Seq#Length(s) }
+axiom (forall s: Seq :: { Seq#Length(s) }
   (Seq#Length(s) == 0 ==> s == Seq#Empty())
 // The following would be a nice fact to include, because it would enable verifying the
 // GenericPick.SeqPick* methods in Test/dafny0/SmallTests.dfy.  However, it substantially
 // slows down performance on some other tests, including running seemingly forever on
 // some.
-//  && (Seq#Length(s) != 0 ==> (exists x: T :: Seq#Contains(s, x)))
+//  && (Seq#Length(s) != 0 ==> (exists x: Box :: Seq#Contains(s, x)))
   );
 
 // The empty sequence $Is any type
-//axiom (forall<T> t: Ty :: {$Is(Seq#Empty(): Seq T, TSeq(t))} $Is(Seq#Empty(): Seq T, TSeq(t)));
+//axiom (forall t: Ty :: {$Is(Seq#Empty(): Seq, TSeq(t))} $Is(Seq#Empty(): Seq, TSeq(t)));
```

Comment:
Likewise

Reply:
Addressed this in [this commit](https://github.com/dafny-lang/dafny/pull/4535/commits/013ffdbc4e3413ecf5e09b10a5d2a4d25d02ac4d).